blog/docs/releases/announcing-knative-v1-1-release.md

```diff
@@ -0,0 +1,172 @@
+- broker.spec.config is now required ([#1555](https://github.com/knative-sandbox/eventing-kafka-broker/pull/1555))
+
+#### 💫 New Features & Changes
+
+- The kafka-controller deployment emits probe requests against the data plane (kafka-sink-receiver and kafka-broker-receiver) to determine Kafka Broker and KafkaSink readiness ([#1495](https://github.com/knative-sandbox/eventing-kafka-broker/pull/1495))
+- You can now configure which header format(b3 multi header, b3 single header, and w3c trace-context) to be used while using Zipkin backend ([#1546](https://github.com/knative-sandbox/eventing-kafka-broker/pull/1546))
+- Handle non-retryable HTTP status codes as reported in the spec:
+For more information: [data-plane.md](https://github.com/knative/specs/blob/c348f501de9eb998b4fd010c54d9127033ee41be/specs/eventing/data-plane.md#event-acknowledgement-and-delivery-retry) ([#1574](https://github.com/knative-sandbox/eventing-kafka-broker/pull/1574))
+
+
+#### RabbitMQ Broker and Source v1.1
+
+<!-- Original notes are here: https://github.com/knative-sandbox/eventing-rabbitmq/releases/tag/knative-v1.1.0 -->
+
+
+#### 💫 New Features & Changes
+
+- Adds tracing with opencensus to the dispatcher and ingress ([#370](https://github.com/knative-sandbox/eventing-rabbitmq/pull/370))
+- Adds tracing with opencensus to the dispatcher and ingress ([#370](https://github.com/knative-sandbox/eventing-rabbitmq/pull/370))
+- Adds tracing with opencensus to the dispatcher and ingress ([#370](https://github.com/knative-sandbox/eventing-rabbitmq/pull/370))
+- Adds tracing with opencensus to the dispatcher and ingress ([#370](https://github.com/knative-sandbox/eventing-rabbitmq/pull/370))
+- RabbitMQ source now can use predeclared queues instead of creating new ones ([#493](https://github.com/knative-sandbox/eventing-rabbitmq/pull/493))
+- Adds tracing with opencensus to the dispatcher and ingress ([#370](https://github.com/knative-sandbox/eventing-rabbitmq/pull/370))
+- Now the source Adapter processes messages concurrently depending on the channel_config.prefetch_count argument (default to 1) ([#522](https://github.com/knative-sandbox/eventing-rabbitmq/pull/522))
+- Adds tracing with opencensus to the dispatcher and ingress ([#370](https://github.com/knative-sandbox/eventing-rabbitmq/pull/370))
+- Do not expose the broker class that the controller operates on as an environment variable ([#512](https://github.com/knative-sandbox/eventing-rabbitmq/pull/512))
+- Adds tracing with opencensus to the dispatcher and ingress ([#370](https://github.com/knative-sandbox/eventing-rabbitmq/pull/370))
+- The RabbitMQ Source now translates RabbitMQ messages according to the RabbitMQ Protocol Binding Spec for CloudEvents ([#475](https://github.com/knative-sandbox/eventing-rabbitmq/pull/475))
+  - Falls back to RabbitMQ message content type if it is not set on the CloudEvent data or headers.
+  - Avoids re-wrapping RabbitMQ messages that are already in the CloudEvent format
```

Comment:
Think the indentation needs to be a bit more (it's not showing up as indented in the preview). 
```suggestion
    - Falls back to RabbitMQ message content type if it is not set on the CloudEvent data or headers.
    - Avoids re-wrapping RabbitMQ messages that are already in the CloudEvent format
```

Reply:
yeah it doesn't ident

Reply:
good catch

Reply:
Now it looks correct with 4 spaces
![image](https://user-images.githubusercontent.com/1094878/150246133-245a7b4d-1af6-45d4-8bd0-c48f782ddb9d.png)